bin/diagnostic.js

```diff
@@ -1 +1,19 @@
 'use strict'
+
+const ask = require('../lib/ask.js')
```

Comment:
Your code is on the right track, you simply need to include this `ask` variable in your function